cli/cmd/plugin/ui/web/tanzu-ui/node-server/src/routes/api/responses/unmanaged.json

```diff
@@ -0,0 +1,13 @@
+[{
+    "name": "Work Space Cluster",
+    "provider": "MiniKube",
+    "status":"Running"
+},{
+    "name": "Tanzu Cluster",
+    "provider": "Kind",
+    "status":"Offline"
+},{
+    "name": "UX/UI Cluster",
```

Comment:
Might be nice to include Docker

Reply:
I think I was told the only two providers would be kind and Minikube, please let me know if this is inaccurate.